leveldb/storage/file_storage.go

```diff
@@ -113,7 +113,7 @@ func OpenFile(path string, readOnly bool) (Storage, error) {
 		logSize:  logSize,
 	}
 	runtime.SetFinalizer(fs, (*fileStorage).Close)
-	return fs, nil
+	return &meteredStorage{s: fs}, nil
```

Comment:
To be consistent with other metrics, the metric should be tied to to DB instance instead of Storage instance. You see if you use `leveldb.Open` instead of `leveldb.OpenFile` convenience func closing DB doesn't necessary closing storage instance also, thus the metrics won't be reset.

Reply:
I'm not sure if you meant to use the wrapper like this: `newSession(storage.IOCounterWrapper(stor), o)` at `leveldb.Open`